src/Mvc/Mvc.Testing/src/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -46,6 +46,16 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="_GeneratePublishTestManifest" BeforeTargets="Publish" DependsOnTargets="_ResolveMvcTestProjectReferences">
+    <ItemGroup>
+      <_ManifestProjects Include="%(_ContentRootProjectReferences.FusionName)">
+        <ContentRoot>$([System.IO.Path]::GetDirectoryName(%(_ContentRootProjectReferences.MSBuildSourceProjectFile)))</ContentRoot>
```

Comment:
Could this be `~`? Then you wouldn't need an extra property in the task.

Reply:
Good point, this used to do something like `~/assemblyname` but now that everything is flat in the base dir, its no longer needed